src/display/api.js

```diff
@@ -1410,10 +1418,12 @@ var PDFWorker = (function PDFWorkerClosure() {
  * @ignore
  */
 var WorkerTransport = (function WorkerTransportClosure() {
-  function WorkerTransport(messageHandler, loadingTask, pdfDataRangeTransport) {
+  function WorkerTransport(messageHandler, loadingTask, pdfDataRangeTransport,
+                           canvasFactory) {
     this.messageHandler = messageHandler;
     this.loadingTask = loadingTask;
     this.pdfDataRangeTransport = pdfDataRangeTransport;
+    this.canvasFactory = canvasFactory;
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99022111, this line should be removed.